experiments/openebs/openebs-target-network-loss/README.md

```diff
@@ -0,0 +1,130 @@
+## Experiment Metadata
+
+<table>
+<tr>
+<th> Type </th>
+<th>  Description  </th>
+<th> Storage </th>
+<th> Application </th>
+<th> K8s Platform </th>
+</tr>
+<tr>
+<td> Chaos </td>
+<td> Inject n/w delay on storage target/controller </td>
+<td> OPENEBS </td>
+<td> Percona MySQL  </td>
+<td> Any </td>
+</tr>
+</table>
+
+## Entry-Criteria
+
+-   Application services are accessible & pods are healthy
+-   Application writes are successful 
+
+## Exit-Criteria
+
+-   Application services are accessible & pods are healthy
+-   Data written prior to chaos is successfully retrieved/read
+-   Database consistency is maintained as per db integrity check utils
+-   Storage target pods are healthy
```

Comment:
@shubhamchaudhary  is this the exit criteria for this test.

Reply:
fixed